test/fixtures/v5/fork/straight_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Mantengase recto en el cruce",
         "fr": "Rester tout droit à l'embranchement",
         "id": "Tetap lurus di pertigaan",
+        "it": "Mantieni la Continua dritto al bivio",
```

Comment:
There is a problem with italian.
With left or right modifier "keep {modifier} at the fork" it's ok.
With "straight" modifier the phrase should be like "{modifier} at the fork" and not "Keep {modifier} at the fork" (translated in italian: "Continua dritto al bivio", without the initial "keep")